src/lib/logMsg/logMsg.h

```diff
@@ -497,7 +497,7 @@ do                                                                     \
                                                                        \
   if ((text = lmTextGet s) != NULL)                                    \
   {                                                                    \
-    lmOut(text, 'F', __FILE__, __LINE__, "***", 0, NULL);              \
+    lmOut(text, 'F', "ForcedLog", 0, "***", 0, NULL);                  \
```

Comment:
This is to have all the LM_F lines aligned vertically in the logfile.
If the name of the file is printed bofore the 'message' and the different file names don't have the exact same number  of characters, it gets unaligned and really hard to read.